dallinger/experiment.py

```diff
@@ -49,6 +49,7 @@ def new_meth(self, *args, **kwargs):
 class Experiment(object):
     """Define the structure of an experiment."""
     app_id = None
+    channel = None
```

Comment:
As we add more attributes here, it'd be nice to add comments explaining what they are used for.